nix/pkgs/default.nix

```diff
@@ -136,6 +136,16 @@ let
       inherit (pkgs) haskell-nix;
     };
 
+  # Collect everything to be exported under `plutus.lib`: builders/functions/utils
+  lib = {
+    ci = pkgs.callPackage ../lib/ci.nix { };
```

Comment:
... and so we probably shouldn't export this either.

Reply:
Yep, makes sense. I dropped that change.